src/js/handler/time/resize.js

```diff
@@ -215,15 +217,22 @@ TimeResize.prototype._updateSchedule = function(scheduleData) {
         newEnds = new TZDate(schedule.getStarts()).addMinutes(30);
     }
 
+    changes = common.getChangesSchedule(
+        schedule,
+        ['end'],
+        {end: newEnds}
+    );
+
     /**
      * @event TimeResize#beforeUpdateSchedule
      * @type {object}
-     * @property {Schedule} schedule - schedule instance to update
-     * @property {Date} start - start time to update
-     * @property {Date} end - end time to update
+     * @property {Schedule} schedule - The original schedule instance
+     * @property {object} changes - end time to update
```

Comment:
The legacy properties' JSDoc must not be removed. Mark it as deprecated. `calendar.js` file is too.

Reply:
right. I'll fix it.